<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des étudiants</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Matricule</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="matricule_search"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_rechercher">
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nom_input"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Prénom</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="prenom_input"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Date de naissance</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="date_naissance_input"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Sexe</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="radio_f">
           <property name="text">
            <string>F</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_h">
           <property name="text">
            <string>H</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Filière</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="filiere_combo">
        <item>
         <property name="text">
          <string>Informatique</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gestion</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mécanique</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Niveau</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="niveau_combo">
        <item>
         <property name="text">
          <string>1ère année</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2ème année</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3ème année</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btn_actualiser">
        <property name="text">
         <string>Actualiser</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ajouter">
        <property name="text">
         <string>Insertion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_modifier">
        <property name="text">
         <string>Modification</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_supprimer">
        <property name="text">
         <string>Suppression</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="table_etudiants">
      <column>
       <property name="text">
        <string>Matricule</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prénom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date de naissance</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sexe</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Filière</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Niveau</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>